ci/build_libtelio.py

```diff
@@ -381,12 +382,7 @@ def main() -> None:
         target_os = "macos" if args.command == "lipo" else args.os
 
         packages = LIBTELIO_CONFIG[target_os].get("packages", None)
-        if (
-            args.debug
-            and not args.tcli
-            and "tcli" in packages
-            and "qnap" not in args.os
-        ):
+        if args.debug and not args.tcli and "tcli" in packages:
```

Comment:
:thinking: not entirely clear what is happening with this change and one below. Could you give a brief?

Reply:
before we were including `tcli` binary in the qnap builds, now we're excluding and replacing it with `teliod` and also removing moose directory from the linker search paths.

We're not planning to use moose in qnap, so this can be removed completely from the config.